packs/_source/items/poison/potion-of-poison.yml

```diff
@@ -117,19 +117,7 @@ system:
         override: false
       damage:
         onSave: half
-        parts:
```

Comment:
Agreed this damage shouldn't be connected to the save, but it should still have a damage activity.